src/leafdoc.js

```diff
@@ -567,7 +576,17 @@ Leafdoc.prototype._stringifyNamespace = function(namespace, isMini) {
 	});
 };
 
-
+/*
+ * 🍂method isDocumentableInheritable(name: String): Boolean
+ * Determines documentable can be inherited.
+ */
+Leafdoc.prototype.isDocumentableInheritable = function(name) {
```

Comment:
What about custom documentables via `registerDocumentable()`? It'd be nice to specify if custom documentables are inheritable or not. Maybe have the inheritable ones in an array.